TPCCAGPUTracking/Merger/AliHLTTPCGMPropagator.h

```diff
@@ -28,6 +28,12 @@ class AliHLTTPCGMPropagator
 {
 public:
 
+   /// Enumeration of field regions
+  enum  FieldRegion : int { 
+    TPC = 0,    ///< TPC
+    TRD = 1,    ///< outer TPC -> outer TRD
```

Comment:
Hi Sergey,

looks good. Just one comment, if you use typed enums, you could use a smaller type, in this case unsigned short.
I'll test it later.

Cheers
David